tests/snippets/test_frame.py

```diff
@@ -563,3 +564,18 @@ def test_types_to_latex() -> None:
     df.to_latex(position='some')
     # caption param was extended to accept tuple in 1.2.0 https://pandas.pydata.org/docs/whatsnew/v1.2.0.html
     df.to_latex(caption=("cap1", "cap2"))
+
+
+def test_types_rename() -> None:
+    class AnyObject:
```

Comment:
`AnyObject = type("AnyObject")` might be shorter

Reply:
Inlined this completely